charts/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.20.1
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.1.2
+version: 4.1.3
```

This is a new feature so I think we should bump the minor version, not the patch version.

cc: @jamengual thoughts?

```suggestion
version: 4.2.2
```

---

cc: @samuelbailey123